### ISSUE TITLE
espeak ng does not speak if there are umlauts in the path to NVDA.exe

### ISSUE THREAD

**Issue by BFW-Wuerzburg** [nvaccess/nvda#19582]
### Brief summary

espeak ng cannot be used as speech output if an umlaut such as ö ä ü is found in the path to NVDA.

### Steps to reproduce

- Create a portable version of NVDA.
- An umlaut like ü ö ä should appear in the path to NVDA.exe.
  "D:\my NVDA\Würzburg\nvda.exe"
- espeak as a voice output does not speak.

### Actual behavior

espeak ng does not speak

### Expected behavior

espeak ng should speak or there should be a warning message that espeak ng will not speak for the path.

### NVDA logs, crash dumps and other attachments

INFO - __main__ (14:22:18.172) - MainThread (15408):
Starting NVDA version 2025.3 x86
INFO - core.main (14:22:18.846) - MainThread (15408):
Config dir: D:\my NVDA\Würzburg\userConfig
INFO - config.ConfigManager._loadConfig (14:22:18.852) - MainThread (15408):
Loading config: D:\my NVDA\Würzburg\userConfig\nvda.ini
INFO - core.main (14:22:21.455) - MainThread (15408):
Windows version: Windows 11 24H2 (10.0.26100.7623) workstation AMD64
INFO - core.main (14:22:21.455) - MainThread (15408):
Using Python version 3.11.9 (tags/v3.11.9:de54cf5, Apr  2 2024, 10:00:00) [MSC v.1938 32 bit (Intel)]
INFO - core.main (14:22:21.455) - MainThread (15408):
Using comtypes version 1.4.11
INFO - core.main (14:22:21.459) - MainThread (15408):
Using configobj version 5.1.0 with validate version 1.0.1
ERROR - synthDriverHandler.setSynth (14:22:22.178) - MainThread (15408):
setSynth failed for espeak
Traceback (most recent call last):
  File "synthDriverHandler.pyc", line 514, in setSynth
  File "synthDriverHandler.pyc", line 477, in getSynthInstance
  File "synthDrivers\espeak.pyc", line 215, in __init__
  File "synthDrivers\_espeak.pyc", line 370, in initialize
OSError: exception: access violation reading 0x0000005C
INFO - synthDriverHandler.setSynth (14:22:22.178) - MainThread (15408):
Searching for next synthDriver
INFO - synthDriverHandler.findAndSetNextSynth (14:22:22.178) - MainThread (15408):
Falling back to next synthDriver silence
INFO - synthDriverHandler.setSynth (14:22:22.182) - MainThread (15408):
Loaded synthDriver silence
WARNING - mathPres.initialize (14:22:22.189) - MainThread (15408):
MathPlayer 4 not available
INFO - core._setUpWxApp (14:22:22.371) - MainThread (15408):
Using wx version 4.2.2 msw (phoenix) wxWidgets 3.2.6 with six version 1.17.0
INFO - brailleInput.initialize (14:22:22.373) - MainThread (15408):
Braille input initialized
INFO - braille.initialize (14:22:22.373) - MainThread (15408):
Using liblouis version 3.34.0
INFO - braille.initialize (14:22:22.373) - MainThread (15408):
Using pySerial version 3.5
INFO - braille.BrailleHandler._setDisplay (14:22:22.375) - MainThread (15408):
Loaded braille display driver 'noBraille', current display has 0 cells.
INFO - brailleDisplayDrivers.freedomScientific.BrailleDisplayDriver.__init__ (14:22:22.546) - ThreadPoolExecutor-0_0 (9640):
Found Focus 40 connected via custom (\\?\usb#vid_0f4e&pid_0114#1234567#{a5dcbf10-6530-11d2-901f-00c04fb951ed})
INFO - braille.BrailleHandler._setDisplay (14:22:22.552) - ThreadPoolExecutor-0_0 (9640):
Loaded braille display driver 'freedomScientific', current display has 40 cells.
INFO - core.main (14:22:22.596) - MainThread (15408):
Java Access Bridge support initialized
INFO - UIAHandler.UIAHandler.MTAThreadFunc (14:22:22.707) - UIAHandler.UIAHandler.MTAThread (8012):
UIAutomation: IUIAutomation6
INFO - core.main (14:22:23.029) - MainThread (15408):
NVDA initialized
INFO - globalCommands.script_navigatorObject_devInfo (14:22:29.479) - MainThread (15408):
Developer info for navigator object:
name: 'Desktop'
role: LIST
processID: 18012
roleText: None
states: FOCUSABLE, FOCUSED
isFocusable: True
hasFocus: True
Python object: <NVDAObjects.IAccessible.sysListView32.List object at 0x07285DF0>
Python class mro: (<class 'NVDAObjects.IAccessible.sysListView32.List'>, <class 'NVDAObjects.IAccessible.List'>, <class 'NVDAObjects.IAccessible.IAccessible'>, <class 'NVDAObjects.window.Window'>, <class 'NVDAObjects.NVDAObject'>, <class 'documentBase.TextContainerObject'>, <class 'baseObject.ScriptableObject'>, <class 'baseObject.AutoPropertyObject'>, <class 'garbageHandler.TrackedObject'>, <class 'object'>)
description: None
location: RectLTWH(left=0, top=0, width=1920, height=1080)
value: None
TextInfo: <class 'NVDAObjects.NVDAObjectTextInfo'>
appModule: AppModule(explorer, appName='explorer', processID=18012)
appModule.productName: 'Betriebssystem Microsoft® Windows®'
appModule.productVersion: '10.0.26100.7689'
appModule.helperLocalBindingHandle: None
appModule.appArchitecture: 'AMD64'
windowHandle: 132196
windowClassName: 'SysListView32'
windowControlID: 1
windowStyle: 1442855488
extendedWindowStyle: 0
windowThreadID: 8344
windowText: 'FolderView'
displayText: ''
IAccessibleObject: <POINTER(IAccessible) ptr=0x5a21f70 at 76a17b0>
IAccessibleChildID: 0
IAccessible event parameters: windowHandle=132196, objectID=-4, childID=0
IAccessible accName: 'Desktop'
IAccessible accRole: ROLE_SYSTEM_LIST
IAccessible accState: STATE_SYSTEM_FOCUSED, STATE_SYSTEM_FOCUSABLE, STATE_SYSTEM_VALID (1048580)
IAccessible accDescription: None
IAccessible accValue: None

### NVDA type

portable copy

### NVDA version

2025.3.2

### Have you tried any other versions of NVDA? If so, please report their behaviors.

2025.*

### Windows version

Windows 11 25h2

### Name and version of other software in use when reproducing the issue

b

### Other information about your system

German system

### Does the issue still occur after restarting your computer?

I have restarted my computer and the issue still occurs

### If NVDA add-ons are disabled, is your problem still occurring?

I have restarted NVDA with add-ons disabled and the issue still occurs

### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?

I have run the COM Registration Fixing Tool and the issue still occurs

**Comment by SaschaCowley** [comment]
See #11392 for the likely solution
